packages/babel-helpers/src/helpers/usingCtx.js

```diff
@@ -0,0 +1,79 @@
+/* @minVersion 7.23.0 */
+
+var disposeSuppressedError2;
+
+export default function _usingCtx() {
+  if (!disposeSuppressedError2) {
+    if (typeof SuppressedError !== "undefined") {
+      // eslint-disable-next-line no-undef
+      disposeSuppressedError2 = SuppressedError;
+    } else {
+      disposeSuppressedError2 = function SuppressedError(suppressed, error) {
+        this.suppressed = suppressed;
+        this.error = error;
+        this.stack = new Error().stack;
+      };
+      Object.setPrototypeOf(disposeSuppressedError2.prototype, Error.prototype);
+    }
+  }
+
+  var empty = {},
+    stack = [];
+  function using(isAwait, value) {
+    if (value != null) {
```

Comment:
I used `v!=null` here because it doesn't look like people would use `document.all` here.